test/JDBC/input/datatypes/TestSpatialFun-vu-prepare.txt

```diff
@@ -0,0 +1,123 @@
+CREATE TABLE YourTableTemp ( ID INT PRIMARY KEY, PointColumn geometry ); 
+
+INSERT INTO YourTableTemp (ID, PointColumn) VALUES (1, geometry::Point(3.0, 4.0, 4326)), (2, geometry::Point(5.0, 6.0, 4326));
+
+CREATE TABLE YourTableTemp1 ( ID INT PRIMARY KEY, STX geometry ); 
+INSERT INTO YourTableTemp1 (ID, STX) VALUES (1, geometry::Point(3.0, 4.0, 4326)), (2, geometry::Point(5.0, 6.0, 4326));
+
+CREATE TABLE YourTableTemp2 ( ID INT PRIMARY KEY, PointColumn1 geometry, PointColumn2 geometry ); 
+INSERT INTO YourTableTemp2 (ID, PointColumn1, PointColumn2) VALUES (1, geometry::Point(3.0, 4.0, 4326), geometry::Point(4.0, 5.0, 4326));
+
+CREATE TABLE TableATemp (ID INT PRIMARY KEY, PointA geometry); 
+CREATE TABLE TableBTemp (ID INT PRIMARY KEY, PointB geometry);
+INSERT INTO TableATemp (ID, PointA) VALUES (1, geometry::Point(1.0, 2.0, 4326)); 
+INSERT INTO TableBTemp (ID, PointB) VALUES (1, geometry::Point(3.0, 4.0, 4326));
+
+CREATE TABLE SPATIALPOINTGEOG_dttemp (location geography)
+
+INSERT INTO SPATIALPOINTGEOG_dttemp (location) VALUES ( geography::STGeomFromText('Point(47.65100 -22.34900)', 4326) )
+
+INSERT INTO SPATIALPOINTGEOG_dttemp (location) VALUES ( geography::STGeomFromText('Point(1.0 2.0)', 4326) )
+
+#Negative Test for STGeomFromText for Geography with SRID 0
```

Comment:
We don't need tests for `STGeomFromText` or other predefined functions as they are already covered earlier in `TestSpatialPoint-*` files

Reply:
resolved